### DISCUSSION TITLE
Typer command can not run a script which imports other original scripts.

### DISCUSSION THREAD

**Discussion by NMZ0429** [tiangolo/typer-cli#142] [Questions] [answered]
First and foremost, thank you so munch for developing this awesome library and I hope I don't bother you by opening this issue.

The problem is that I can not use typer-cli command to run a function if it's importing other python scripts.

Given the file structure and contents, the command `typer main.py run` fails.
```
.
├── main.py
├── helloworld.py
```

**main.py**
```python
import typer
from helloworld import print_hello


def greet():
    print_hello()


if __name__ == "__main__":
    typer.run(greet)
```

**helloworld.py**
```python
def print_hello():
    print("Hello World")
```

**output**
```shell
$typer main.py run
...
ModuleNotFoundError: No module named 'helloworld'
```

**Comment by vinigfer** [comment]
I'm also having this issue. Did you end up finding a solution for it @NMZ0429 ?

**Comment by yohay13** [comment]
✋

**Comment by tiangolo** [comment, marked as answer]
Hello! This repo, `typer-cli`, is now integrated into `typer`, when you install with `pip install typer` you get the `typer` command with everything. This is since Typer 0.12.1. :rocket: 

I just tried your example with the latest version and it all seems to work well! :tada: 

If it doesn't, please let me know more details in a question in Typer: https://github.com/tiangolo/typer/discussions/new?category=questions
